robottelo/ui/contentsearch.py

```diff
@@ -0,0 +1,146 @@
+# -*- encoding: utf-8 -*-
+
+"""Implements Content Search UI"""
+
+from robottelo.ui.base import Base, UIError
+from robottelo.ui.locators import locators
+
+
+class ContentSearch(Base):
+    """Manipulates Content Search from UI"""
+
+    ft_locator = None
+
+    def select_filter_type(self, filter_type):
+        """Specify content type that you like to choose from 'Content' dropdown
+        At that moment we support next types:
+        'Content Views'
+        'Products'
+        'Repositories'
+        'Packages'
+        'Puppet Modules'
+        In the same time, it is necessary to keep order like in list above
+        (Content Views->Products->Repositories->Packages, Puppet Modules)
+        because in Content Search UI the lower you go in list more elements
+        will be added to panel
```

Comment:
Thanks. 10 years learning English and have that feeling that I will never get to necessary level :)